### PR TITLE
Fixed formatting for gitlab example

### PR DESCRIPTION
**Pull request by fchorney**
Forgot to escape the quotes and add semi-colons for the GitLab examples.

### CODE REVIEW

**Comment by fchorney** [comment]
This possibly doesn't actually require a version bump actually

File: Project.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name = "CompatHelper"
 uuid = "aa819f21-2bde-4658-8897-bab36330d9b7"
 authors = ["Dilum Aluthge", "Brown Center for Biomedical Informatics", "contributors"]
-version = "3.0.0"
+version = "3.1.0"
```
Comment (mattBrzezinski) [inline Project.toml:4]: Since it's a bug fix.

```suggestion
version = "3.0.1"
```

Comment (fchorney) [reply Project.toml:4]: I guess it's a README bug fix

**Comment by fchorney** [comment]
bors try

**Review by mattBrzezinski — APPROVED** [review]

**Pull request merged**